### PR TITLE
bump gradle version to 7.3

### PR DESCRIPTION
**Pull request by fremartini**
closing #107 

as correctly stated by @jonasanker updating the gradle version fixed the warnings related to 'mapping new ns (...) to old ns (...)'

### CODE REVIEW

**Comment by marfavi** [comment]
Works on my machine 👍🏽

File: android/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,5 @@
-#Fri Jun 23 08:50:38 CEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3-bin.zip
```
Comment (jonasanker) [inline android/gradle/wrapper/gradle-wrapper.properties:3]: What's the difference between the all and bin version of gradle?

Comment (jonasanker) [reply android/gradle/wrapper/gradle-wrapper.properties:3]: Also. Can we align the versions, so they use the same version of 7.x?

**Review by jonasanker — COMMENTED** [review]
Comments on versioning

File: android/gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (fremartini) [reply android/gradle/wrapper/gradle-wrapper.properties:3]: @jonasanker it seems they are independent. Looking through the gradle settings in Android Studio it seems a new version of gradle was released yesterday (7.4) 

image

Comment (jonasanker) [reply android/gradle/wrapper/gradle-wrapper.properties]: All good then. I'll approve

**Review by jonasanker — APPROVED** [review]

**Comment by marfavi** [comment]
Squash these commits? :)

**Pull request merged**